meinberlin/apps/topicprio/views.py

```diff
@@ -96,80 +94,73 @@ class Meta:
         fields = ['category']
 
 
-class TopicMgmtView(ProjectContextDispatcher,
-                    DashboardBaseMixin,
-                    rules_mixins.PermissionRequiredMixin,
-                    filter_views.FilteredListView):
+class TopicListDashboardView(mixins.DashboardComponentMixin,
+                             mixins.DashboardBaseMixin,
+                             mixins.DashboardContextMixin,
+                             filter_views.FilteredListView):
     model = models.Topic
-    template_name = 'meinberlin_topicprio/topic_mgmt_list.html'
+    template_name = 'meinberlin_topicprio/topic_dashboard_list.html'
     filter_set = TopicCreateFilterSet
     permission_required = 'a4projects.add_project'
     project_url_kwarg = 'slug'
```

Comment:
there is no longer a valid project_url_kwarg
the list view is routed with the module_slug as named parameter as defined in .dashboard.py